content/docs/dvclive/api-reference/live/log_params.md

```diff
@@ -0,0 +1,54 @@
+# Live.log_params()
+
+Logs the given group of parameters `params`, saving them as yaml.
```

Comment:
Not sure it's worth pointing out the yaml format at the top, although making it clear somewhere in this doc makes sense.